controllers/articles.controllers.js

```diff
@@ -0,0 +1,13 @@
+const fetchArticleById = require('../models/articles.models')
+
+getArticleById = (req, res) => {
+    const { article_id } = req.params
+    fetchArticleById(article_id).then((article) => {
+        res.status(200).send({article: article})
+    })
+    .catch((err) => {
```

Comment:
I would like you to do something more productive with your errors, such as passing them onto an error handler